content/SCALE/GettingStarted/FirstTimeLogin.md

```diff
@@ -140,11 +140,11 @@ Each warning level has a different icon and color to express its urgency. To mak
 
 ### Alert Services
 
-The *Alert Services* screen has options to create and edit alert services. The *Alert Services* screen displays existing alert services in a list that users can filter by *Type*, *Level*, and *Enabled*.
+The **Alert Services** screen has options to create and edit alert services. The **Alert Services** screen displays existing services in a list that users can filter by *Type*, *Level*, and *Enabled*.
```

Comment:
if the emphasis is to indicate a UI option or selection use bold. if the emphasis is added to a general discussion of a filter type verses the specific UI, don't use italics or bold or capitalize.
```suggestion
The **Alert Services** screen has options to create and edit alert services. The **Alert Services** screen displays existing services in a list that users can filter by **Type**, **Level**, and **Enabled**.
```